build.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+root=$(cd "$(dirname "$0")"; pwd -P)
+artifacts=$root/artifacts
+configuration=Release
+
+restorePackages=0
+skipTests=0
+
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
+        -\?|-h|--help)
+            echo "./build.sh [--skip-tests]"
+            exit 1
+            ;;
+
+        --skip-tests)
+            skipTests=1
+            ;;
+
+        *)
+            __UnprocessedBuildArgs="$__UnprocessedBuildArgs $1"
+            ;;
+    esac
+
+    shift
+done
+
+export CLI_VERSION="2.0.0"
```

Comment:
Should use 2.0.3 now.

Reply:
Probably needs a complimentary update in `Build.ps1` too - this is just an SDK update, doesn't need any target framework changes or anything else updating.

Reply:
When the CLI version is `2.0.3` does the install script url also change?
There is no `Build.ps1` file. It's not needed yet.